Makefile

```
@@ -106,40 +111,31 @@ psql-r:  ## Remote connect to postgis, uses $POSTGRES_USER and $POSTGRES_DB.
 pre-commit-a:  ## Check all files with pre-commit.
 	pre-commit run --all-files
 
-.PHONY: distclean
-distclean: clean uninstall  ## Clean everything and uninstalls.
-
-.PHONY: clean
-clean: clean-frontend clean-backend clean-mdbook clean-storybook clean-scraper  ## Clean everything.
-
-.PHONY: clean-frontend
-clean-frontend:  ## Remove /frontend node_modules .
-	cd frontend && rm -rf node_modules
+.PHONY: docker-up
+docker-up:  ## Start a containerized dev environment
+	cd .devcontainer && docker compose -p "permaplant_devcontainer" up
 
-.PHONY: clean-backend
-clean-backend:  ## Remove /backend/target .
-	cd backend && make clean
+# Install/Clean
 
-.PHONY: clean-mdbook
-clean-mdbook:  ## Remove /mdbook .
-	mdbook clean
-
-.PHONY: clean-scraper
-clean-scraper:  ## Remove /scraper/node_modules .
-	cd scraper && rm -rf node_modules
+.PHONY: install
+install:  $(FAKE_FILE)  node_modules  ## Install ALL dependencies within the source repo.
```

> package.json somehow doesn't work. The other install with creating a .install file work fine.

What seems to be the problem? I actually thought that must be trivial?

Following you tried:

```suggestion
install:  $(FAKE_FILE)  frontend/package.json  ## Install ALL dependencies within the source repo.
```

Did you already consider to also have a frontend Makefile? In general we should look out to keep everything consistent. Can't we name `node_modules` to something like `install-node-modules`?

> Also simplified it a bit.

Always great :heart: